packages/cdktf-cli/templates/java/cdktf.json

```diff
@@ -0,0 +1,5 @@
+{
+  "language": "java",
+  "app": "mvn -e -q compile exec:java",
+  "terraformProviders": ["aws@~> 2.0"]
```

Comment:
Only slightly related to this PR: I was thinking that it could make sense to introduce a naming schema for the various templates. Rather than making AWS the default for `typescript`, `python` and `java`, I think we could default to a minimal config without any providers or modules pre defined. There could be provider specific templates if we want to. What do you think?

Reply:
I completely agree about defaulting to no providers/modules. It adds quite a bit of time for something that may not be relevant, especially with pre-built providers available.

I think it makes sense to only maintain 1 template per language and let others fall to #339

Reply:
I'd like to get the pre-built providers created before removing default provider. I started some of that work [here](https://github.com/terraform-cdk-providers/cdktf-provider-project/pull/14).

Reply:
That's quite cool 👍

Reply:
> I completely agree about defaulting to no providers/modules. It adds quite a bit of time for something that may not be relevant, especially with pre-built providers available.
> I think it makes sense to only maintain 1 template per language and let others fall to #339

Let's do this in a separate PR